src/langapi/Makefile

```diff
@@ -1,4 +1,4 @@
-SRC = mode.cpp language_ui.cpp languages.cpp language_util.cpp
```

Comment:
See https://gcc.gnu.org/onlinedocs/gcc/Vague-Linkage.html for more info and why it should still be ok on most platforms. To me this means: avoid it, unless you have very good reasons.